react/react.py

```diff
@@ -96,7 +98,7 @@ def to_json(data):
         :return: Received data in json format
         :rtype: str
         """
-        if isinstance(data, str):
+        if isinstance(data, six.string_types):
```

Comment:
> Nevermind, `six.string_types` is `str` in py3. Don't change it.

Uhm. On a second thought, passing `bytes` in py3 should have the same behaviour as passing `str` in py2, so yes, it should check for `six.text_type` :stuck_out_tongue: . However, I don't know why someone would pass a string (bytes/str or str/unicode), honestly.